### PR TITLE
fix(dependabot): fix dependabot warnings

### PR DESCRIPTION
**Pull request by qbiqing**
While the dependabot bug with the PR separator is not yet fixed, this PR consolidates the dependabot package upgrades.
Ref: https://github.com/dsaidgovsg/registrywatcher/pull/50#issuecomment-1118120178

### CODE REVIEW

File: ui/yarn.lock [hunk 1]
```diff
@@ -5157,11 +5157,16 @@ flush-write-stream@^1.0.0:
...
-follow-redirects@^1.0.0, follow-redirects@^1.10.0:
+follow-redirects@^1.0.0:
   version "1.14.8"
   resolved "https://registry.yarnpkg.com/follow-redirects/-/follow-redirects-1.14.8.tgz#016996fb9a11a100566398b1c6839337d7bfa8fc"
   integrity sha512-1x0S9UVJHsQprFcEC/qnNzBLcIxsjAV905f/UkQxbclCsoTWlacCNOpQa/anodLl2uaEKFhfWOvM2Qg77+15zA==
 
+follow-redirects@^1.14.0:
+  version "1.15.0"
+  resolved "https://registry.yarnpkg.com/follow-redirects/-/follow-redirects-1.15.0.tgz#06441868281c86d0dda4ad8bdaead2d02dca89d4"
+  integrity sha512-aExlJShTV4qOUOL7yF1U5tvLCB0xQuudbf6toyYA0E/acBNw71mvjFTnLaRp50aQaYocMR0a/RMMBIHeZnGyjQ==
```
Comment (guangie88) [inline ui/yarn.lock:5168]: It differs from the one from dependabot:
https://github.com/dsaidgovsg/registrywatcher/pull/56/files#diff-3a968206d6de2fecfc5dacd7d94bab7744c9f5d5c999a816164d95cbc135c316R5160

Is this intentional?

Comment (qbiqing) [reply ui/yarn.lock:5168]: This results after running `yarn install` after making the dependabot changes. Might be dependency changes that are required?

File: testutils/testutils.go [hunk 2]
```diff
@@ -151,7 +151,7 @@ func (helper *TestHelper) StartRegistry() (string, string, error) {
 				Name: "always",
 			},
 		},
-		nil, "")
```
Comment (guangie88) [inline testutils/testutils.go:154]: v17 vs v20: additional `platform *specs.Platform` param as breaking change:
https://pkg.go.dev/github.com/docker/docker@v17.12.1-ce-rc1+incompatible/client#Client.ContainerCreate
https://pkg.go.dev/github.com/docker/docker@v20.10.14+incompatible/client#Client.ContainerCreate

File: ui/yarn.lock (hunk 1 above)
Comment (qbiqing) [reply ui/yarn.lock]: Ran `yarn install--frozen-lockfile` as suggested, the additional lines were not generated, hence the original changes from dependabot would also work

Comment (guangie88) [reply ui/yarn.lock]: Thanks for trying, `yarn` and `npm` seems notorious for changing lock file as default behaviour without flags haha.

**Review by guangie88 — APPROVED** [review]
LGTM!

**Pull request merged**